app/helpers/users_helper.rb

```diff
@@ -0,0 +1,7 @@
+module UsersHelper
+  def gravatar_for(user)
```

Comment:
You should add a comment here explaining where this code came from.  This gives future developers context on how this code was developed and gives credit to whomever posted it originally.

e.g. "See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration" in spec_helper.rb.